Lib/asyncio/selector_events.py

```diff
@@ -274,9 +274,8 @@ def _ensure_fd_no_transport(self, fd):
     def _add_reader(self, fd, callback, *args):
         self._check_closed()
         handle = events.Handle(callback, args, self, None)
-        try:
-            key = self._selector.get_key(fd)
-        except KeyError:
+        key = self._selector.get_map().get(fd)
```

Comment:
I didn't check for `get_map` being `None` (ie selector closed) since the only way that should happen is if the selector is closed and these are already guarded by `self._check_closed()` and the only time the selector is closed is in `.close()` AFAICT